tests/modules/programs/thunderbird/thunderbird.nix

```diff
@@ -41,6 +43,10 @@
     # Disable warning so that platforms' behavior is the same
     darwinSetupWarning = false;
 
+    # Darwin doesn't support wrapped Thunderbird, using unwrapped instead;
+    # using -latest- because ESR is currently broken on Darwin
+    package = realPkgs.thunderbird-latest-unwrapped;
+
```

Comment:
Is this needed for anything or can we just let it use the stubbed package?

Reply:
`symlinkJoin` that creates native messaging hosts directory assumes it's a directory and fails

```
> @thunderbird@: No such file or directory
```